functions/assignOfflineContact.ts

```diff
@@ -0,0 +1,141 @@
+import '@twilio-labs/serverless-runtime-types';
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+import {
+  responseWithCors,
+  bindResolve,
+  error400,
+  error500,
+  success,
+  send,
+} from '@tech-matters/serverless-helpers';
+// eslint-disable-next-line prettier/prettier
+import type { PromiseValue } from 'type-fest';
+
+const TokenValidator = require('twilio-flex-token-validator').functionValidator;
+
+type EnvVars = {
+  TWILIO_WORKSPACE_SID: string;
+  TWILIO_CHAT_TRANSFER_WORKFLOW_SID: string;
+};
+
+export type Body = {
+  targetSid?: string;
+  finalTaskAttributes: string;
+};
+
+// eslint-disable-next-line prettier/prettier
+type TaskInstance = PromiseValue<ReturnType<ReturnType<ReturnType<ReturnType<Context['getTwilioClient']>['taskrouter']['workspaces']>['tasks']>['fetch']>>;
+// eslint-disable-next-line prettier/prettier
+type WorkerInstance = PromiseValue<ReturnType<ReturnType<ReturnType<ReturnType<Context['getTwilioClient']>['taskrouter']['workspaces']>['workers']>['fetch']>>;
```

Comment:
Sorry this ugly types, couldn't pull them from Twilio libs and found this the easiest way to use them.